modules/network-observability-flows-format.adoc

```diff
@@ -0,0 +1,246 @@
+// Automatically generated by 'hack/asciidoc-flows-gen.sh'. Do not edit.
+:_content-type: REFERENCE
+[id="network-observability-flows-format-specifications_{context}"]
+= Network Flows format specifications
+
+The document is organized in two main categories: _Labels_ and regular _Fields_. This distinction only matters when querying Loki. This is because _Labels_, unlike _Fields_, must be used in link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
+
+Outside of Loki context, for instance when reading this specification as a reference for the Kafka export feature, you must ignore this distinction about _Labels_ and _Fields_, and consider all of them at the same level in the JSON structure.
+
+
+== Labels
+
+
+SrcK8S_Namespace::
+
+• `Optional` *SrcK8S_Namespace*: `string`
```

Comment:
what does "Optional" mean here? Is that field may not be present or it could have "" (empty) or null values?

Reply:
@jotak ^^ Joel should know, as he generated this file for me.

Reply:
It may not be present